client/my-sites/guided-transfer/host-credentials-page/index.jsx

```diff
@@ -3,13 +3,28 @@
  */
 import React, { PropTypes, Component } from 'react';
 import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
 import Bluehost from './bluehost';
 import SiteGround from './siteground';
+
```

Comment:
It was intentional - I'd intended to separate the sub-components from the library components. You're right though, it's probably unnecessary. Removed in eccc2e2ab0d2c91f9f6e4d9ce68d438c05a5d8d0